__tests__/components/ModalWrapper.test.js

```diff
@@ -1,7 +1,8 @@
 // Copyright 2019 Stanford University see LICENSE for license
 
 import React from 'react'
-import { render } from '@testing-library/react'
+import { cleanup, render } from '@testing-library/react'
+import { setupModal } from '../test_utilities/testUtils'
```

Comment:
This should just be: `import { setupModal } from 'testUtils'`. I'm going to fix.